rust/theoros/src/main.rs

```diff
@@ -42,13 +70,25 @@ async fn main() -> Result<()> {
 
     init_tracing(&config.app_name, LOG_LEVEL)?;
 
+    let governor_conf = Arc::new(GovernorConfigBuilder::default().per_second(4).burst_size(2).finish().unwrap());
+    let governor_limiter = governor_conf.limiter().clone();
+    // a separate background task to clean up
+    std::thread::spawn(move || loop {
```

Comment:
This should probably live in `services/api/mod.rs`?

And we add `join_set.spawn(...)` for the 2 tasks created here, since they're related to the api.

Reply:
fixed